### PR TITLE
Add .travis.yml

### PR DESCRIPTION
**Pull request by merlinND**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+install: true
+language: scala
```
Comment (merlinND) [inline .travis.yml:2]: I don't like this

Comment (sygi) [inline .travis.yml:2]: For me, it looks fine.

Comment (merlinND) [inline .travis.yml:2]: Thanks @sygi :ferris_wheel:

**Comment by merlinND** [comment]
:shipit:

**Pull request merged**